asciidoc-markup-sample-doc/asciidoc-markup-samples.adoc

```diff
@@ -164,7 +166,7 @@ a|
 ....
 kbd:[key1(+key2+...)]
```

Comment:
I think the middle column should show the mark-up that gets rendered in the right one.

Reply:
Ah, ok, no problem. Wait a min.

Reply:
Fixed.